commons-hpcc/src/main/java/org/hpccsystems/commons/errors/HpccErrorCode.java

```diff
@@ -0,0 +1,20 @@
+package org.hpccsystems.commons.errors;
+
+public enum HpccErrorCode implements IErrorCode
+{
+
+    UNCAUGHT_EXCEPTION(200), INFO(300);

```

Comment:
this list of codes feels incomplete. Do we know which hpcc errors we're representing here so we can get the full list from the hpcc code?
for example dfu error codes: https://github.com/hpcc-systems/HPCC-Platform/blob/ff2b7df781c6205b12795e131751b1eecf14fbc4/dali/dfu/dfuerror.hpp

Reply:
This is a very small set. The intention was that wsclient and dfsclient would implement IErrorCode with their own wsclient/dfsclient-specific error codes; this class is here as an example/base class to extend.